charts/publisher/templates/common/configmap.yaml

```diff
@@ -2,29 +2,31 @@ apiVersion: v1
+  ASSETS_PREFIX: "{{ .Values.common.env.assetsPrefix }}"
+  BASIC_AUTH_USERNAME: "{{ .Values.common.env.basicAuthUsername }}"
+  DEFAULT_TTL: "{{ .Values.common.env.defaultTtl }}"
+  EMAIL_GROUP_BUSINESS: "{{ .Values.common.env.emailGroupBusiness }}"
+  EMAIL_GROUP_CITIZEN: "{{ .Values.common.env.emailGroupCitizen }}"
+  EMAIL_GROUP_FORCE_PUBLISH_ALERTS: "{{ .Values.common.env.emailGroupForcePublishAlerts }}"
+  FACT_CHECK_SUBJECT_PREFIX: "{{ .Values.common.env.factCheckSubjectPrefix }}"
+  FACT_CHECK_USERNAME: "{{ .Values.common.env.factCheckUsername }}"
+  GOVUK_APP_DOMAIN: "{{ .Values.common.env.govukAppDomain | default (printf "%s.eks.%s.%s" .Release.Namespace .Values.common.govukEnvironment .Values.common.govukDomainInternal) }}"
+  GOVUK_APP_DOMAIN_EXTERNAL: "{{ .Values.common.govukDomainExternal | default (printf "%s.eks.%s.%s" .Release.Namespace .Values.common.govukEnvironment .Values.common.govukDomainExternal) }}"
+  GOVUK_APP_NAME: "{{ .Values.common.env.govukAppName | default .Release.Name }}"
+  GOVUK_APP_ROOT: "{{ .Values.common.env.govukAppRoot }}"
+  GOVUK_APP_TYPE: "{{ .Values.common.env.govukAppType }}"
+  GOVUK_ASSET_ROOT: "{{ .Values.common.env.govukAssetRoot | default (printf "https://assets.%s.eks.%s.%s" .Release.Namespace .Values.common.govukEnvironment .Values.common.govukDomainExternal) }}"
+  ASSET_HOST: "{{ .Values.common.env.assetHost | default (printf "https://www-origin.%s.eks.%s.%s" .Release.Namespace .Values.common.govukEnvironment .Values.common.govukDomainExternal) }}"
+  GOVUK_ENVIRONMENT: "{{ .Values.common.govukEnvironment }}"
+  GOVUK_WEBSITE_ROOT: "{{ .Values.common.env.govukWebsiteRoot | default (printf "https://www-origin.%s.eks.%s.%s" .Release.Namespace .Values.common.govukEnvironment .Values.common.govukDomainExternal) }}"
+  PLEK_SERVICE_CONTENT_STORE_URI: "{{ .Values.common.env.plekServiceContentStoreUri | default (printf "http://content-store.%s.%s" .Release.Namespace .Values.common.govukDomainInternal) }}"
+  PLEK_SERVICE_DRAFT_ORIGIN_URI: "{{ .Values.common.env.plekServiceContentStoreUri | default (printf "https://draft-origin-%s.eks.%s.%s" .Release.Namespace .Values.common.govukEnvironment .Values.common.govukDomainExternal) }}"
+  PLEK_SERVICE_PUBLISHING_API_URI: "{{ .Values.common.env.plekServicePublishingApiUri | default (printf "http://publishing-api-web.%s.%s" .Release.Namespace .Values.common.govukDomainInternal) }}"
+  PLEK_SERVICE_SIGNON_URI: "{{ .Values.common.env.plekServiceSignonUri | default (printf "https://signon-%s.eks.%s.%s" .Release.Namespace .Values.common.govukEnvironment .Values.common.env.govukDomainExternal) }}"
+  PLEK_SERVICE_STATIC_URI: "{{ .Values.common.env.plekServiceStaticUri | default (printf "http://static.%s.%s" .Release.Namespace .Values.common.govukDomainInternal) }}"
+  PORT: "{{ .Values.web.port }}"
+  RAILS_ENV: "{{ .Values.common.env.railsEnv | default .Values.common.govukEnvironment }}"
```

Comment:
Somewhat confusingly, the right value for RAILS_ENV is `production` in basically all cases (except when doing local development where you want live-reloading on code changes and so on). Defaulting to the same value as govukEnvironment would cause all sorts of interesting and exciting issues 😀